client/my-sites/post-type-list/empty-content.jsx

```diff
@@ -0,0 +1,66 @@
+/**
+ * External dependencies
+ */
+import React, { PropTypes } from 'react';
+import { connect } from 'react-redux';
+
+/**
+ * Internal dependencies
+ */
+import localize from 'lib/mixins/i18n/localize';
+import { getPostType } from 'state/post-types/selectors';
+import { getSelectedSiteId } from 'state/ui/selectors';
+import { getEditorPath } from 'state/ui/editor/selectors';
+import QueryPostTypes from 'components/data/query-post-types';
+import EmptyContent from 'components/empty-content/empty-content';
+
+function PostTypeListEmptyContent( { siteId, translate, status, typeObject, editPath } ) {
+	let title, action;
+
+	if ( 'draft' === status ) {
+		title = translate( 'You don\'t have any drafts.' );
+	} else if ( typeObject ) {
+		title = translate( 'You don\'t have any %s.', {
+			args: [ typeObject.label.toLocaleLowerCase() ]
+		} );
+	}
+
+	if ( typeObject ) {
+		action = translate( 'Start a %s', {
+			args: [ typeObject.labels.singular_name ]
+		} );
+	}
+
+	return (
+		<div>
+			{ siteId && (
+				<QueryPostTypes siteId={ siteId } />
+			) }
+			<EmptyContent
+				title={ title }
+				line={ translate( 'Would you like to create one?' ) }
+				action={ action }
+				actionURL={ editPath }
+				illustration="/calypso/images/pages/illustration-pages.svg"
+				illustrationWidth={ 150 } />
+		</div>
+	);
+}
+
+PostTypeListEmptyContent.propTypes = {
+	translate: PropTypes.func,
+	type: PropTypes.string,
+	status: PropTypes.string,
+	typeObject: PropTypes.object,
+	editPath: PropTypes.string
```

Comment:
It is a rare event for me to find something your PRs to comment about :laughing: